vadovas/spinta.rst

```diff
@@ -669,6 +669,73 @@ __ https://cx-oracle.readthedocs.io/en/latest/index.html
     $ spinta inspect -r sql oracle+cx_oracle://user:pass@host:port/db -o sdsa.xlsx
 
 
+XML
+===
+
+Jei naudojamas XML duomenų šaltinis, :term:`DSA` struktūrą galima sugeneruoti dviem būdais:
+iš `XML` formatu turimų duomenų arba iš `XSD` schemos.
+
+XML failas
+----------
+
+Iš XML formatu turimų duomenų :term:`DSA` generuojamas naudojant komandą `inspect`:
+
+.. code-block:: sh
+
+    $ spinta inspect -r xml data.xml -o data_xml.xlsx
+
+Šiuo atveju, jei `XML` struktūra bus ši:
+
+.. code-block:: xml
+
+    <city>
+        <name>
+            Kaunas
+        </name>
+        <population>
+            200000
+        </population>
+    </city>
+
+Tai sugeneruotas :term:`DSA` atrodys taip:
+
+.. code-block:: sh
+
+    id | d | r | b | m | property     | type    | ref | source     | prepare | level | access | uri | title | description
+       | dataset                      |         |     |            |         |       |        |     |       |
+       |   | resource                 | xml     |     | data.xml   |         |       |        |     |       |
+       |                              |         |     |            |         |       |        |     |       |
+       |   |   |   | City             |         |     | /city      |         |       |        |     |       |
+       |   |   |   |   | name[]       | string  |     | name       |         |       |        |     |       |
+       |   |   |   |   | population[] | integer |     | population |         |       |        |     |       |
```

Comment:
Ar tikrai sugeneruojamas variantas su `[]`? Jei taip, tai čia klaida generatoriuje ir turėtu būti be `[]`.

Reply:
taip, šitą jau užregistravau: https://github.com/atviriduomenys/spinta/issues/633 

Pataisysiu čia, parašysiu taip, kaip turi būti.